app/components/subcase/subcase-titles-panel/subcase-titles-panel-view.js

```diff
@@ -14,33 +14,35 @@ export default class SubcaseTitlesPanelView extends Component {
   @service currentSession;
 
   @tracked approved; // or acknowledged
+  @tracked isClosed;
```

Comment:
Similar to previous comments, you can define a `get isFinalMeeting() { ... }` and use that instead of introducing the tracked property `isClosed`.